lib/fastlane/plugin/wpmreleasetoolkit/helper/github_helper.rb

```diff
@@ -28,6 +31,16 @@ def self.get_milestone(repository, release)
         return mile
       end
 
+      # Fetch all the PRs for a given milestone
+      #
+      # @param String repository The repository name (including the organization) [ex: wordpress-mobile/wordpress-ios]
+      # @param String milestone The name of the tag we're downloading from [ex: 16.9]
```

Comment:
Still wrong description for this `milestone` param 😛

Also, not sure that YARD properly parses the doc when types are not specified between square brackets


```suggestion
      # @param [String] repository The repository name, including the organization (e.g. `wordpress-mobile/wordpress-ios`)
      # @param [String] milestone The name of the milestone we want to fetch the list of PRs for (e.g.: `16.9`)
```

Reply:
> not sure that YARD properly parses the doc when types are not specified between square brackets
TIL – addressed in 8c3e4affab2acde6067ba4af4de8d044591a350d